app/Http/Controllers/API/ConferencesController.php

```diff
@@ -7,24 +7,17 @@
 use Carbon\Carbon;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Auth;
+use App\Http\Requests\StoreConferenceRequest;
 
 class ConferencesController extends Controller
 {
-    public function store(Request $request)
+    public function store(StoreConferenceRequest $request)
     {
-        $after = (new Carbon($request->input('start_date')))->subDay();
```

Comment:
Haha, I saw those, but that part of the talk made no sense. That only matters if you're unit testing your controllers, which we don't do here.

Gonna go look at it again and think about it some more.. :)